documentation/src/main/asciidoc/topics/endpoint_interop.adoc

```diff
@@ -216,24 +216,25 @@ If you plan to store entries in the cache as custom Java objects or POJOs, you m
 
 To make entity classes available to the server, do the following:
 
-* Create a `JAR` file that contains the entities and dependencies.
-* Stop {brandname} if it is running.
+. Create a JAR file that contains the entities and dependencies.
+. Stop {brandname} if it is running. {brandname} only loads entity classes at
+boot time.
+. Copy the JAR to the `server/lib` directory of your {brandname} server installation.
 +
-{brandname} loads entity classes during boot. You cannot make entity classes available to {brandname} if the server is running.
+----
+├── server
+│   ├── lib
+│       ├── deployment.my-entities.jar
+----
++
+. Specify the `JAR` file as a module in the Cache Manager configuration.
```

Comment:
I'm not sure if this is still valid. @tristantarrant can you confirm?

Reply:
the can be removed. it is no longer valid. The server doesn't start with it.

Reply:
@pruivo can you please double check this and see if my changes are OK? thanks for looking into it.

Reply:
they look fine!